src/api/grainConfig.js

```diff
@@ -4,19 +4,24 @@ import {type DistributionPolicy} from "../core/ledger/applyDistributions";
 import * as G from "../core/ledger/grain";
 import * as C from "../util/combo";
 import * as NullUtil from "../util/null";
+import {toDiscount} from "../core/ledger/grainAllocation";
 
 export type GrainConfig = {|
   +immediatePerWeek: number,
   +balancedPerWeek: number,
+  +recentPerWeek: number,
+  +discount?: number,
```

Comment:
for clarity, let's rename to `recentPerWeekTimeDecayRate` or something. otherwise it's not at all clear what `discount` does, or that it's connected to the `recentPerWeek`

Reply:
Agreed. I implemented name change, but stuck with `discount` in the actual `RecentPolicy` type.  I think this confusion is indicative that there's probably a clearer way to do this.  I like the idea of adding policy "tickets".
```js
{
  policies: [
    "BALANCED": 0
    "IMMEDIATE": 0,
    "RECENT": {
       perWeek: 10,
       discount: 0.5
    }
  ],
  "maxSimultaneousDistributions": 100
]
```
or
```js
{
  policies: [
    {
       policyType: "BALANCED",
       perWeek: 10
    },
    {
       policyType: "IMMEDIATE",
       perWeek: 10
    },
    {
       policyType: "RECENT",
       perWeek: 10,
       discount: 0.5
    },
  ],
  "maxSimultaneousDistributions": 100
}
```
I also am strongly biased towards not having to include each policy as mandatory in the config.  This does not scale well.